src/core_modules/capture-core/metaDataStoreLoaders/trackedEntityTypes/loadTrackedEntityTypes.js

```diff
@@ -1,50 +1,58 @@
 // @flow
+import { chunk } from 'capture-core-utils';
 import { queryTrackedEntityTypesOutline } from './queries';
 import { storeTrackedEntityTypes } from './quickStoreOperations';
 
-function getOptionSetsOutline(trackedEntityTypesOutline) {
-    const optionSetsOutline = trackedEntityTypesOutline.reduce((accOptionSets, type) => {
-        const optionSetsForType = type.trackedEntityTypeAttributes &&
-            type.trackedEntityTypeAttributes.reduce((accOptionSetsForType, TETA) => {
-                if (TETA.trackedEntityAttribute && TETA.trackedEntityAttribute.optionSet) {
-                    accOptionSetsForType.push(TETA.trackedEntityAttribute.optionSet);
-                }
-                return accOptionSetsForType;
-            }, []);
-
-        if (optionSetsForType) {
-            accOptionSets = [...accOptionSets, ...optionSetsForType];
-        }
-
-        return accOptionSets;
-    }, []);
-    return optionSetsOutline;
-}
-
-
-export async function loadTrackedEntityTypes(
-) {
-    const trackedEntityTypesOutline = await queryTrackedEntityTypesOutline();
-    const optionSetsOutline = getOptionSetsOutline(trackedEntityTypesOutline);
-    const { convertedData: loadedTrackedEntityTypes } = await storeTrackedEntityTypes();
-
-    let trackedEntityAttributeIds = [];
-    if (loadedTrackedEntityTypes) {
-        trackedEntityAttributeIds = loadedTrackedEntityTypes.reduce((accAttributeIds, TET) => {
-            if (TET.trackedEntityTypeAttributes) {
-                const attributeIds = TET
+const loadTrackedEntityTypesBatch = async (idBatch: Array<string>) => {
+    const { convertedData: loadedTrackedEntityTypes = [] } = await storeTrackedEntityTypes(idBatch);
```

Comment:
This is like a flashback here. hehe

Is this name really describing what we are doing here? Sorry to bring this up again but to me this name needs to be worked out still.